.github/workflows/version.yml

```diff
@@ -0,0 +1,51 @@
+name: Bump Versions
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  version-packages:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          cache: yarn
+          node-version: v20.7.0
+
+      - name: node_modules cache
+        uses: actions/cache@v2
+        with:
+          path: "**/node_modules"
+          key: ${{ runner.os }}-node-modules-cache-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-node-modules-cache-
+
+      - name: Install dependencies
+        run: yarn install --immutable
```

Comment:
Should we have a check after this to make sure that `yarn.lock` was not modified?

Reply:
That's what `--immutable` does. It's like `npm install --ci`.